modules/client-configuration/pages/clients-raspberrypios.adoc

```diff
@@ -0,0 +1,214 @@
+====
+
+[IMPORTANT]
+====
+You need all the new channels fully synchronized before bootstrapping any {raspberrypios} client.
+====
+
+endif::[]
+
+== Check Synchronization Status
+
+ifeval::[{suma-content} == true]
+
+include::snippets/check_sync_webui_suma.adoc[]
+
+endif::[]
+
+ifeval::[{uyuni-content} == true]
+
+include::snippets/check_sync_webui_uyuni.adoc[]
+
+endif::[]
+
+include::snippets/check_sync_cli.adoc[]
+
+
+[NOTE]
+====
+{raspberrypios} channels can be very large.
+Synchronization can sometimes take several hours.
+====
+
+
+== Create an Activation Key
+
+You need to create an activation key that is associated with your {raspberrypios} channels.
+
+For more information on activation keys, see xref:client-configuration:activation-keys.adoc[].
+
+
+
+
+== Manage GPG Keys
+
+Clients use GPG keys to check the authenticity of software packages before they are installed.
+Only trusted software can be installed on clients.
+
+[IMPORTANT]
+====
+Trusting a GPG key is important for security on clients.
+It is the task of the administrator to decide which keys are needed and can be trusted.
+Because a software channel cannot be used when the GPG key is not trusted, the decision of assigning a channel to a client depends on the decision of trusting the key.
+====
+
+For more information about GPG keys, see xref:client-configuration:gpg-keys.adoc[].
+
+[NOTE]
+====
+{raspberrypios} clients can require multiple GPG keys to be installed.
+====
+
+When synchronizing third-party {raspberrypios} repositories, you will need to import the appropriate GPG key on the server.
+If the GPG key is missing, synchronization will fail.
+
+For {raspberrypios} repositories, only the metadata is signed.
+Therefore importing a GPG key for the software channel is not needed.
+Packages will not be re-signed by {productname}.
+
+To see which GPG keys are already imported to {productname} Server, run the following command:
+
+----
+sudo gpg --homedir /var/lib/spacewalk/gpgdir --list-keys
+----
+
+To import a new GPG key, use the `--import` parameter:
+
+----
+sudo gpg --homedir /var/lib/spacewalk/gpgdir --import <filename>.gpg
+----
+
+
+
+== Root Access
+
+The root user on {raspberrypios} is disabled by default for SSH access.
+
+To be able to onboard using a regular user, you need to edit the [filename]``sudoers`` file.
+
+.Procedure: Granting Root User Access
+
+. On the client, edit the [filename]``sudoers`` file:
++
+----
+sudo visudo
+----
++
+Grant [command]``sudo`` access to the user by adding this line at the end of the [filename]``sudoers`` file.
+Replace [systemitem]``<user>`` with the name of the user that is bootstrapping the client in the {webui}:
++
+----
+<user>  ALL=NOPASSWD: /usr/bin/python, /usr/bin/python2, /usr/bin/python3, /var/tmp/venv-salt-minion/bin/python
+
+
+
+== Register Clients
+
+include::snippets/create_bootstrap_repo_register.adoc[]
```

Comment:
This is not rendering correctly:

![image](https://github.com/uyuni-project/uyuni-docs/assets/4226070/cafcb545-cfc5-48da-8dd2-0ab880dc3746)